keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/VerificationEvaluator.kt

```diff
@@ -10,12 +11,19 @@ interface VerificationEvaluator<VERIFICATION: Verification> {
   val supportedVerification: Pair<String, Class<VERIFICATION>>
 
   /**
+   * @param metadata as returned by [start].
    * @return the current status of the verification.
    */
-  fun evaluate() : VerificationStatus
+  fun evaluate(
+    context: VerificationContext,
+    verification: Verification,
+    metadata: Map<String, Any?>
+  ): VerificationStatus
 
   /**
    * Start running [verification].
+   *
+   * @return any metadata needed to [evaluate] the verification in future.
    */
-  fun start(verification: Verification)
+  fun start(context: VerificationContext, verification: Verification): Map<String, Any?>
```

Comment:
The `start` method can now return metadata (e.g. orca task id) that will get persisted and then passed to future calls to `evaluate`.